### DISCUSSION TITLE
Will v4 be able to support Firefox?

### DISCUSSION THREAD

**Discussion by sxdjt** [bluecaret/carettab#355] [Help and Support] [answered]
I have version 3.6.2 (the last one that worked on Firefox), and I would gladly subscribe to have the latest version available.

Is the v4 framework extensible to Firefox?

Thanks for the massive update, by the way...

**Comment by bluecaret** [comment, marked as answer]
Unfortunately Firefox is still not supported on the newer versions. However! I have plans to try and work on this soon, v4 should make it easier to support Firefox, I'm hoping. I can't guarantee anything here, but I would like to support Firefox again and am looking into it. Thanks!

**Reply by krzdabrowski** [reply]
Hey, any progress on supporting Firefox?

**Reply by bluecaret** [reply]
@krzdabrowski I did make an attempt on it this week but ran into some bugs still that seem problematic to fix that I didn't have time to look into. It gives me hope that it's a possibility though. That being said I've not been as active on development lately as I'm working on other projects so I'm sorry to say I'm still not guaranteeing anything here. Thank you for your continued interest however, it helps :)

**Reply by sxdjt** [reply]
@bluecaret - Thanks for looking at it!  I would be lost without it, even the old version on FF!
